openedx/features/lums/models.py

```diff
@@ -0,0 +1,31 @@
+"""
+Models for LUMS custom Information.
+"""
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfileExtension(models.Model):
+    """
+    This model is used to store additional custom profile fields.
+    """
+
+    PUNAJB = 'punjab'
+    SINDH = 'sindh'
+    KPK = 'kpk'
+    BALOCHISTAN = 'balochistan'
+
+    PROVINCE_CHOICES = (
+        (PUNAJB, _('Punjab')),
+        (SINDH, _('Sindh')),
+        (KPK, _('Khyber Pakhtunkhwa')),
+        (BALOCHISTAN, _('Balochistan')),
+    )
```

Comment:
do we want to have `Gilgit-Baltistan`?

Reply:
No. The product owner specifically asked to add only 4 provinces.